projects/widgets/docs/linken-van-artikels-aan-events-via-widgets.md

```diff
@@ -0,0 +1,123 @@
+
+![artikel-bruzz-op-uiv.png](../assets/images/artikel-bruzz-op-uiv.png)
+
+## Hoe gaat dat precies in z'n werk?
+
+Om een artikel aan een evenement in UiTdatabank te linken dien je een specifieke widget `<script>` tag in je artikel op te nemen, met daarin de unieke identifier van het evenement waarvoor het artikel gaat (`cdbid`).
+
+> Je kan dit soort `<script>` tag enkel integreren mits toestemming van publiq. Wil je hiermee aan de slag? Contacteer ons dan op <partnerships@publiq.be>.
+
+### 1. Het `<script>` tag integreren in een artikel
+
+Het `<script>` tag dat je dient te integreren in een artikel ziet er ongeveer als volgt uit:
+
+```html
+<div id="embed" data-cdbid="ffe4f207-2f8d-4dce-87a6-08abf59aefbe">
+  <script type="text/javascript" src="https://projectaanvraag-api.uitdatabank.be/widgets/layout/1234.js"></script>
+</div>
+```
+
+Dit `<script>` tag bevat 2 variabelen:
+
+1. `data-cdbid`: dit bepaalt welk evenement aan het artikel gelinkt wordt en moet door de redacteur (via het CMS van de website) toegevoegd worden
+2. `src`: dit is de referentie naar de unieke JavaScript file van je widget, die we jou bezorgen.
+
+Bovenstaande embed-code genereert in het artikel een blok met daarin de praktische informatie van het evenement: de naam, locatie, het tijdstip en de prijs van het evenement.
+
+### 2. Het unieke ID van een evenement vinden
+
+De waarde van `data-cdbid` uit het `<script>`tag bepaalt welk evenement ingeladen wordt. In het voorbeeld van hierboven is dit het evenement met cdbid `ffe4f207-2f8d-4dce-87a6-08abf59aefbe`.
+
+Een redacteur hoeft dus enkel te kiezen welk evenement hij wilt integreren in zijn artikel, door de waarde voor `data-cdbid` aan te passen. De waarde voor `data-cdbid` kan je terugvinden op verschillende manieren.
+
+* **in de agenda van UiTinVlaanderen**
+
+Je kan naar de detailpagina van het evenement op UiTinVlaanderen surfen. In de url vind je het `cdbid` van het evenement:
+
+![redactionele-tools-detail-uiv.png](../assets/images/redactionele-tools-detail-uiv.png)
+
+Dit `cdbid` dien je vervolgens te kopiëren en te gebruiken als waarde voor `data-cdbid` in het widget `<script>` tag.
+
+* **in de eigen UiT-agenda**
+
+Je kan naar de detailpagina van het evenement in de eigen UiT-agenda surfen. In de url vind je het `cdbid` van het evenement:
+
+![redactionele-tools-detail-bruzzwebsite.png](../assets/images/redactionele-tools-detail-bruzzwebsite.png)
+
+Dit `cdbid` dien je vervolgens te kopiëren en te gebruiken als waarde voor `data-cdbid` in het widget `<script>` tag.
+
+* **door de UiTdatabank te doorzoeken vanuit je eigen CMS**
+
+Je kan ook een zoek bouwen doorheen evenementen vanuit je eigen CMS.
```

Comment:
I assume that this would be done using Search API? Maybe we should specify that? (I'm assuming the video would show an example of how an integration would look like in the own CMS, but not how to build the integration?)